src/zenml/integrations/aws/__init__.py

```diff
@@ -37,8 +38,9 @@ class AWSIntegration(Integration):
     # does not pin it. They fixed this in a later version, so we can probably
     # remove this once we update the sagemaker version.
     REQUIREMENTS = [
-        "boto3==1.21.0",
-        "sagemaker==2.82.2",
+        "boto3==1.26.26",
```

Comment:
Just FYI, this makes the PR a breaking change. I still think we should do this, but as far as I know updating the sagemaker version also allows us to remove the explicit `protobuf>=3.1,<4.0.0` dependency (and the comment above this list). This needs some testing though

Reply:
Did we verify this?

Reply:
Yes. New versions are good to go.